app/jobs/invoices/provider_taxes/pull_taxes_and_apply_job.rb

```diff
@@ -3,8 +3,12 @@
 module Invoices
   module ProviderTaxes
     class PullTaxesAndApplyJob < ApplicationJob
+      include ConcurrencyThrottlable
```

Comment:
Here as well:

Do we need to restrict it to only 5 concurrent calls? If not we can remove this line.
Anrok doesn't have concurrency restrictions, only 10 requests per second per API key.

Reply:
removed